### PR TITLE
Fixed a bug in vmi creation

### PR DESCRIPTION
**Pull request by DanaOrr**
Signed-off-by: Dana Orr <dorr@redhat.com>

## 📝 Description
Fixed a bug in vmi creation

## 🎥 Demo
image
image

### CODE REVIEW

**Review by glekner — COMMENTED** [review]
Great job.
I dont think we need `useActiveNamespace` here.

You should implement this like `VirtualMacineEmptyState`
it receives the URL as a prop from `VirtualMachineList` which has access to namespace`
then you can leave it as ``/k8s/ns/${namespace || 'default'}/templatescatalog`;`

just inject the namespace from `VirtualMachineInstancesList` to `VirtualMacineInstancesEmptyState`

File: src/views/virtualmachinesinstance/list/components/VirtualMachineInstanceEmptyState/VirtualMachineInstanceEmptyState.tsx [hunk 1]
```diff
@@ -17,17 +16,20 @@ import {
...
   const history = useHistory();
-  const namespace = useActiveNamespace();
-  const catalogURL = `/k8s/ns/${namespace || 'default'}/templatescatalog`;
 
   return (
     <EmptyState variant={EmptyStateVariant.large}>
       <EmptyStateIcon icon={VirtualMachineIcon} />
       <Title headingLevel="h4" size="lg">
-        {t('No VirtualMachinesInstaces found')}
+        {t('No VirtualMachinesInstances found')}
```
Comment (yaacov) [inline src/views/virtualmachinesinstance/list/components/VirtualMachineInstanceEmptyState/VirtualMachineInstanceEmptyState.tsx:32]: VirtualMachinesInstances -> VirtualMachineInstances

https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachineinstance

**Pull request merged**